### DISCUSSION TITLE
Is three dimension table in Mainim?

### DISCUSSION THREAD

**Discussion by shuhao-hub** [ManimCommunity/manim#3758] [Q&A]
I wanna make a video about three dimension dynamic programming problem. So I wanna know is the feature in Mainim. I need you help. thank u.

**Comment by uwezi** [comment]
As you can easily check in the documentation of Manim there is no such thing as a pre-made 3D-table. However, I have difficulties to even imagine how a 3D-table should look like, be constructed or rendered. 

You can easily place objects in a regular grid in 3D, but it will be quite a challenge to visualize the arrangement since the elements will (or should) block each other.

I suggest you come over to Discord and discus your plan in a bit more detail - perhaps someone there can help you.
https://docs.manim.community/en/stable/faq/general.html?highlight=discord#where-can-i-find-more-resources-for-learning-manim

**Reply by shuhao-hub** [reply]
Oh! thank you for your reply! I get it.

**Reply by shuhao-hub** [reply]
Maybe, I can use a cube that consists of small cubes in regular grid in 3D to show the process of solving the problem. It's not a big deal if elements block each other.
